src/components/Callout.astro

```diff
@@ -0,0 +1,50 @@
+---
+import callout_889 from '../assets/images/callout_889.svg'
+import callout_map from '../assets/images/callout_map.svg'
+import callout_forum from '../assets/images/callout_forum.svg'
+---
+<section class="sp_callout">
+    <div class="grid-container margin-y-6">
+        <div class="grid-row grid-gap ">
+          <div class="tablet:grid-col-6">
+            <img
+             src ={callout_889}
+             alt ="Three panels from a screen, the first panel has a magnifying glass and reads use the 889 search to find your vendor, the second panel has a green check mark and reads in compliance with Section 889, and third panel has a red exclamation mark and reads not in compliance with Section 889."
+            >
+          </div>
+          <div class="tablet:grid-col-6">
+            <h2>Section 889 Search</h2>
+            <p>Our Section 889 search tool makes it easy to check an entity’s Section 889 status in <a href="https://sam.gov/">SAM.gov</a>.</p>
+            <p><a href="https://889.smartpay.gsa.gov/">Go to the 889 search tool →</a></p>
+          </div>
+        </div>
+        <div class="grid-row grid-gap margin-y-2">
+            <div class="tablet:grid-col-6">
+                <img
+                src ={callout_map}
+                alt ="A map of the contiguous United States with Utah highlighted in a darker color than the rest of the states, with Alaska, Hawaii, and U.S territories also shown."
+                >
+            </div>
+            <div class="tablet:grid-col-6">
+                <h2>State Tax Information</h2>
+                <p>Check the tax-exempt status and policies of U.S. states and territories.</p>
+                <p><a href="/statetax/">Go to tax information →</a></p>
```

Comment:
I think we will need to put `${import.meta.env.BASE_URL}` in front of any links starting with `/` otherwise the links won't work for federalist previews. Alternatively, we could just do relative links: `href="statetax/"`.